packages/yourloops/components/dropdown/basic-dropdown.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2022, Diabeloop
+ * Switch role from caregiver to HCP dialog - Accept terms
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import { FormControl, FormGroup, FormHelperText, InputLabel, MenuItem, Select } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+type SetState<T> = React.Dispatch<React.SetStateAction<T>>;
+type SelectChangeEvent = React.ChangeEvent<{ name?: string; value: unknown }>;
+type HandleChange<E> = (event: E) => void;
+
+export interface Errors {
+  notAllowedValue: boolean;
+}
+
+export interface BasicDropdownProps<T> {
+  onSelect: (value: T) => void;
+  defaultValue: string;
+  disabledValues: string[];
+  values: string[];
+  translationKey: string;
+}
+
+const dialogStyles = makeStyles(
+  () => {
+    return {
+      formControl: {
+        display: "flex",
+      },
+    };
+  },
+);
```

Comment:
Eventually, since it's a very simple CSS, can be shorten with
```suggestion
const dialogStyles = makeStyles(() => ({
  formControl: { display: "flex" },
}), { name: "component-basic-dropdown" });
```